[VI] Fixes and updates in the `Rules` and `Help Centre` articles (#14711)
ppy/osu-wiki: wiki/Help_centre/Silences/vi.md

Author: ```suggestion
Một người dùng có một im lặng đang có hiệu lực không thể làm những điều sau đây:
```

Author: ```suggestion
- Đăng bình luận bất kỳ nơi nào trên website (beatmap, danh sách thay đổi, tin tức).
```

Reviewer: ```
trang website -> website
```

Author: ```suggestion
- Tham gia vào [thảo luận beatmap](/wiki/Beatmap_discussion).
```

Author: ```suggestion
- **Lạm dụng chữ in hoa**: Sử dụng toàn bộ chữ cái in hoa trong trò chuyện
```

Author: ```suggestion
- **Nội dung không mong muốn**: Các website chứa nội dung vi phạm bản quyền, gây hoảng sợ, liên kết giới thiệu và các thứ tương tự
```

Author: ```suggestion
- Sử dụng [trò chuyện](/wiki/Client/Interface/Chat_console) trực tuyến (công khai và riêng tư) cả trong game lẫn trên website.
```

Reviewer: ```
trang website -> website
```

  - mute
  - timeout
  - cấm trò chuyện
---

# Sự im lặng

> **Im lặng** là một thuật ngữ được sử dụng để mô tả hình phạt giới hạn phương tiện giao tiếp của người dùng trong cộng đồng osu!. Các thành viên của [nhóm điều hành toàn cầu](/wiki/People/Global_Moderation_Team) và [nhóm đánh giá đề cử](/wiki/People/Nomination_Assessment_Team) thường đưa ra hình phạt im lặng này để duy trì một môi trường an toàn, cả trên wedsite lẫn trong game.

Author: ```suggestion
**Im lặng** là một thuật ngữ được sử dụng để mô tả hình phạt giới hạn phương tiện giao tiếp của người dùng trong cộng đồng osu!. Các thành viên của [nhóm điều hành toàn cầu](/wiki/People/Global_Moderation_Team) và [nhóm đánh giá đề cử](/wiki/People/Nomination_Assessment_Team) thường đưa ra hình phạt im lặng này để duy trì một môi trường lành mạnh, cả trên website lẫn trong game.
```

  - mute
  - timeout
  - cấm trò chuyện
---

# Sự im lặng

**Im lặng** là một thuật ngữ được sử dụng để mô tả hình phạt giới hạn phương tiện giao tiếp của người dùng trong cộng đồng osu!. Các thành viên của [nhóm điều hành toàn cầu](/wiki/People/Global_Moderation_Team) và [nhóm đánh giá đề cử](/wiki/People/Nomination_Assessment_Team) thường đưa ra hình phạt im lặng này để duy trì một môi trường lành mạnh, cả trên website lẫn trong game.

## Những hạn chế

::: Infobox
![](img/silence-profile.png?1 "Lịch sử im lặng trong trang hồ sơ")
:::

Một người dùng có một im lặng đang có hiệu lực không thể làm những điều sau đây:

- Sử dụng [trò chuyện](/wiki/Client/Interface/Chat_console) trực tuyến (công khai và riêng tư) cả trong game lẫn trên website.

> - Đăng bài trên bất kỳ các diễn đàn của osu!

Reviewer: ```suggestion
- Đăng bài trên bất kỳ các diễn đàn của osu!.
```

- Đăng bình luận bất kỳ nơi nào trên wedsite (beatmap, danh sách thay đổi, tin tức).
- Tham gia vào [thảo luận beatmap](/wiki/Beatmap_discussion).
- Chỉnh sửa chi tiết hồ sơ, thay đổi avatar, ảnh bìa và trang người dùng.

> - Gửi và cập nhật beatmap

Reviewer: ```suggestion
- Gửi và cập nhật beatmap.
```

![](img/silence-ingame.png "Cảnh báo im lặng trong trò chuyện")
:::

Một khi lệnh im lặng được áp dụng cho người dùng, toàn bộ tin nhắn của họ trong trò chuyện đều bị xóa bỏ và một thông báo sẽ được hiển thị trong vị trí ô nhập tin nhắn chứa thời lượng còn lại của im lặng đang hoạt động.[^chat-cleanup]

Khi được áp dụng, im lặng sẽ được hiển thị công khai trong tab `trạng thái tài khoản` của hồ sơ người dùng bị im lặng, và vẫn tiếp tục hiển thị trong 28 ngày sau khi kết thúc. Nếu im lặng hiện tại đang hoạt động, thời lượng của nó cũng sẽ được hiển thị trong tab này. Và khi người dùng có 3 im lặng trở lên được hiển thị, hoặc nếu có một im lặng hiển thị kéo dài hơn 43 giờ, một cảnh báo sẽ xuất hiện trên hồ sơ người dùng. 

## Thời lượng

Thời lượng của một im lặng bắt đầu từ 5 phút và thường gấp đôi với mỗi vi phạm sau đó, tối đa là 28 ngày, nhưng thời gian ban đầu có thể thay đổi tùy thuộc vào vi phạm đã thực hiện và lịch sử của người dùng trước đó. Các yếu tố được xem xét khi xác định độ dài của một im lặng bao gồm:

- **Bầu không khí**: Tình trạng của môi trường trò chuyện hiện tại.
- **Tần suất vi phạm**: sự lặp lại bất chấp những lần im lặng trước đó được áp dụng.

> - **Lịch sử**: bản ghi của những vi phạm trước đó

Reviewer: ```suggestion
- **Lịch sử**: bản ghi của những vi phạm trước đó.
```

Reviewer: viết in hoa chữ cái đầu hết đi ô ._. 3 dòng 41, 42, 43

> Một khi lệnh im lặng được áp dụng cho người dùng, tất cả các tin nhắn của họ trong trò chuyện đều bị xóa bỏ và một thông báo sẽ được hiển thị trong vị trí ô nhập tin nhắn chứa thời gian của im lặng đang hoạt động.[^chat-cleanup]

Author: ```suggestion
Một khi lệnh im lặng được áp dụng cho người dùng, toàn bộ tin nhắn của họ trong trò chuyện đều bị xóa bỏ và một thông báo sẽ được hiển thị trong vị trí ô nhập tin nhắn chứa thời lượng còn lại của im lặng đang hoạt động.[^chat-cleanup]
```

> Khi được áp dụng, im lặng sẽ được hiển thị công khai dưới tab `trạng thái tài khoản` của hồ sơ người dùng bị im lặng, và vẫn tiếp tục hiển thị trong 28 ngày sau khi kết thúc. Nếu im lặng hiện tại đang hoạt động, thời lượng của nó cũng sẽ được hiển thị trong tab này. Và khi người dùng có 3 hoặc thêm im lặng được hiển thị, hoặc nếu có một im lặng hiển thị kéo dài hơn 43 giờ, một cảnh báo về trạng thái hồ sơ của người dùng cũng được hiển thị. 

Author: ```suggestion
Khi được áp dụng, im lặng sẽ được hiển thị công khai trong tab `trạng thái tài khoản` của hồ sơ người dùng bị im lặng, và vẫn tiếp tục hiển thị trong 28 ngày sau khi kết thúc. Nếu im lặng hiện tại đang hoạt động, thời lượng của nó cũng sẽ được hiển thị trong tab này. Và khi người dùng có 3 im lặng trở lên được hiển thị, hoặc nếu có một im lặng hiển thị kéo dài hơn 43 giờ, một cảnh báo sẽ xuất hiện trên hồ sơ người dùng. 
```

## Thời lượng

> Thời lượng của một im lặng bắt đầu từ 5 phút và thường gấp đôi với mỗi vi phạm sau đó, tối đa đến 28 ngày, nhưng thời gian ban đầu có thể thay đổi tùy thuộc vào vi phạm đã thực hiện và lịch sử của người dùng trước đó. Các yếu tố được xem xét khi xác định độ dài của một im lặng tính vào:

Author: ```suggestion
Thời lượng của một im lặng bắt đầu từ 5 phút và thường gấp đôi với mỗi vi phạm sau đó, tối đa là 28 ngày, nhưng thời gian ban đầu có thể thay đổi tùy thuộc vào vi phạm đã thực hiện và lịch sử của người dùng trước đó. Các yếu tố được xem xét khi xác định độ dài của một im lặng bao gồm:
```

- **Bầu không khí**: Tình trạng của môi trường trò chuyện hiện tại.
- **Tần suất vi phạm**: sự lặp lại bất chấp những lần im lặng trước đó được áp dụng.
- **Lịch sử**: bản ghi của những vi phạm trước đó
- **Mức độ nghiêm trọng**: sự nghiêm trọng của hành vi đã thực hiện.

Trong một số các trường hợp, người dùng có thể nhận được một cảnh báo cuối cùng trước khi [hạn chế tài khoản](/wiki/Help_centre/Account_restrictions) được áp dụng, cung cấp một cơ hội để ngừng lại hành vi vi phạm.

## Những lý do phổ biến cho im lặng

> Một số các lý do phổ biến tại sao người dùng nhận được im lặng trong trò chuyện cộng đồng bao gồm, nhưng không giới hạn:

Author: ```suggestion
Một số lý do phổ biến tại sao người dùng nhận được im lặng trong trò chuyện cộng đồng bao gồm, nhưng không giới hạn:
```

## Kháng cáo

> Trong khi các hình thức im lặng được áp dụng để duy trì một môi trường cộng đồng tích cực, đôi khi lỗi cũng có thể xảy ra. Nếu bạn tin rằng hình thức im lặng áp dụng cho bạn là lỗi hoặc không công bằng, vui lòng liên hệ [đội ngũ hỗ trợ tài khoản](/wiki/People/Account_support_team#accounts@ppy.sh) tại [accounts@ppy.sh](mailto:accounts@ppy.sh) và giải thích tình huống. bạn cũng có thể yêu cầu giải thích rõ ràng về lý do im lặng của bạn.

Author: ```suggestion
Trong khi các hình thức im lặng được áp dụng để duy trì một môi trường cộng đồng tích cực, đôi khi lỗi cũng có thể xảy ra. Nếu bạn tin rằng hình thức im lặng áp dụng cho bạn là lỗi hoặc không công bằng, vui lòng liên hệ [đội ngũ hỗ trợ tài khoản](/wiki/People/Account_support_team#accounts@ppy.sh) tại [accounts@ppy.sh](mailto:accounts@ppy.sh) và giải thích tình huống. Bạn cũng có thể yêu cầu giải thích rõ ràng về lý do im lặng của bạn.
```

> Hãy nhớ rằng bạn bắt buộc sử dụng địa chỉ email liên quan với tài khoản osu! của bạn và đề cập đến tên người dùng của bạn để xác nhận đó là bạn.

Author: ```suggestion
Hãy nhớ sử dụng địa chỉ email liên quan với tài khoản osu! của bạn và đề cập đến tên người dùng của bạn để xác nhận đó là bạn.
```

> Hãy nhớ rằng bạn bắt buộc sử dụng địa chỉ email liên quan với tài khoản osu! của bạn và đề cập đến tên người dùng của bạn để xác nhận đó là bạn.

Reviewer: ```suggestion
Hãy nhớ rằng bạn bắt buộc phải sử dụng địa chỉ email liên kết với tài khoản osu! của bạn và đề cập tên người dùng của bạn để xác nhận đó là bạn.
```

## Cách nào để tôi tránh bị im lặng ?

> Đọc kĩ và theo [các quy tắc](/wiki/Rules), [Những cân nhắc về nội dung trực quan](/wiki/Rules/Visual_content_considerations), [Quy định nội dung bài hát](/wiki/Rules/Song_content_rules), và [quy tắc ứng xử với modding và mapping](/wiki/Rules/Code_of_conduct_for_modding_and_mapping). Nếu bạn không chắc liệu điều bạn muốn làm hoặc nói có vi phạm quy định hay không, hãy hỏi [một thành viên của nhóm điều hành toàn cầu](https://osu.ppy.sh/groups/4). 

Reviewer: ```suggestion
Đọc kĩ và tuân thủ [các quy tắc](/wiki/Rules), [Những cân nhắc về nội dung hình ảnh](/wiki/Rules/Visual_content_considerations), [Quy định nội dung bài hát](/wiki/Rules/Song_content_rules), và [quy tắc ứng xử với modding và mapping](/wiki/Rules/Code_of_conduct_for_modding_and_mapping). Nếu bạn không chắc liệu điều bạn muốn làm hoặc nói có vi phạm quy định hay không, hãy hỏi [một thành viên của nhóm điều hành toàn cầu](https://osu.ppy.sh/groups/4). 
```

Reviewer 2: nội dung trực quan là bao gồm mấy cái hiển thị trên trang wed á ông, ko riêng gì hình ảnh đâu

> Ngoài ra, nếu bạn có bài đăng trên diễn đàn cũ, beatmap, bình luận, hoặc những thứ tương tự mà bạn cho rằng chứa nội dung không phù hợp, bạn cũng nên đơn giản là xóa đi hoặc chỉnh sửa phần nội dung nếu có thể. Nếu bạn không thể chỉnh sửa hoặc xóa nội dung vì bất kỳ lý do nào, liên hệ một thành viên của nhóm điều hành toàn cầu để xóa bỏ chúng.

Author: ```suggestion
Ngoài ra, nếu bạn có bài đăng trên diễn đàn cũ, beatmap, bình luận, hoặc những thứ tương tự mà bạn cho rằng chứa nội dung không phù hợp, bạn cũng nên đơn giản là xóa hoặc chỉnh sửa phần nội dung đó nếu có thể. Nếu bạn không thể chỉnh sửa hoặc xóa nội dung vì bất kỳ lý do nào, liên hệ một thành viên của nhóm điều hành toàn cầu để họ xóa nội dung đó.
```
